Chat App/View/Pub/PubChatRoomViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  PubChatRoomViewController.swift
+//  Chat App
+//
+//  Created by Amiel Jireh Cordova on 12/9/20.
+//  Copyright © 2020 Cordova, Jireh. All rights reserved.
+//
+
+import UIKit
+import MessageKit
+import InputBarAccessoryView
+
+class PubChatRoomViewController: MessagesViewController {
+    
+    private var messagesThread: [Message] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.isNavigationBarHidden = false
+        
+        self.title = Constants.bannerLabel
+        self.navigationItem.setHidesBackButton(true, animated: false)
+        
+        maintainPositionOnKeyboardFrameChanged = true
+        messageInputBar.delegate = self
+        messageInputBar.inputTextView.tintColor = .black
+        messageInputBar.inputTextView.placeholder = "Start a new message"
+        messageInputBar.inputTextView.backgroundColor = Constants.messageInputBackground
+        messageInputBar.inputTextView.layer.cornerRadius = 10.0
+        messageInputBar.sendButton.backgroundColor = .darkGray
+        messageInputBar.sendButton.setTitle("send", for: .normal)
+        messageInputBar.sendButton.setTitleColor(.white, for: .normal)
+        messageInputBar.sendButton.layer.cornerRadius = 10.0
+
+        messagesCollectionView.messagesDataSource = self
+        messagesCollectionView.messagesLayoutDelegate = self
+        messagesCollectionView.messagesDisplayDelegate = self
+        
+        let layout = messagesCollectionView.collectionViewLayout as? MessagesCollectionViewFlowLayout
+        layout?.setMessageIncomingAvatarSize(.zero)
+        layout?.setMessageOutgoingAvatarSize(.zero)
+        layout?.setMessageOutgoingMessageBottomLabelAlignment(LabelAlignment(textAlignment: .right, textInsets: UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 8)))
```

Comment:
This exceeds the 120 characters limit per line. You can import swiftlint to avoid this https://github.com/realm/SwiftLint